src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,92 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationServiceImpl service;
+    private StorageDaoImpl storageDao;
+    private User userDefault;
+    private User userFailLogin;
+    private User userFailPassword;
+    private User userFailAge;
+
+    @BeforeEach
+    void setUp() {
+        service = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+
+        userDefault = createUserForTest("Denys","12345678",23);
+        userFailLogin = createUserForTest("Ma","asdasdd34",21);
+        userFailPassword = createUserForTest("Mak","45",21);
+        userFailAge = createUserForTest("Mak","asdqwert6",17);
+    }
+
+    private User createUserForTest(String login, String pass, int age) {
+        User user = new User();
+        user.setLogin(login);
+        user.setPassword(pass);
+        user.setAge(age);
+        storageDao.add(user);
+        return user;
+    }
+
+    @AfterEach
+    void afterEachClearStorage() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_nullUser_notOk() {
```

Comment:
I don't see such naming for other tests as I asked in my previous review

Reply:
I don't understand how I should rename the name of the tests

Reply:
pattern: methodName_specialSituation/event_(ok/notOk)